compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -528,6 +532,12 @@ impl<'value, 'interner> Display for ValuePrinter<'value, 'interner> {
                 let trait_ = self.interner.get_trait(*trait_id);
                 write!(f, "{}", trait_.name)
             }
+            Value::TraitImpl(trait_impl_id) => {
+                let trait_impl = self.interner.get_trait_implementation(*trait_impl_id);
+                let trait_impl = trait_impl.borrow();
+                // TODO: what's a good display for a trait impl?
+                write!(f, "{}", trait_impl.ident)
```

Comment:
I think we should probably display the "signature" of the impl.
E.g. `impl<A, B> Foo for Bar<A, B> where A: Foo`.

Not sure if we store all of the generics so it may have to be close enough if not.